src/scss/ucf-spotlight.scss

```diff
@@ -0,0 +1,180 @@
+// Copied from Athena
+@import 'variables';
+@import 'mixins';
+
+$screen-md-min: 992px;
+$screen-lg-min: 1200px;
+
+// Common to all three layouts
+.spotlight-horizontal,
+.spotlight-vertical,
+.spotlight-square {
+
+  .spotlight-header,
+  .spotlight-copy,
+  .spotlight-link {
+    font-family: $font-family-sans-serif;
+  }
+
+  .spotlight-copy {
+
+    .spotlight-link {
+      color: #337ab7;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+  }
+
+  .btn {
+    @include button-size($btn-padding-y, $btn-padding-x, $font-size-base, $btn-border-radius);
+    @include transition($btn-transition);
+
+    border: $input-btn-border-width solid transparent;
+    border-width: $btn-border-width;
+    display: inline-block;
+    font-size: $btn-font-size;
+    font-weight: $btn-font-weight;
+    letter-spacing: $letter-spacing-base;
+    line-height: $btn-line-height;
+    text-align: center;
+    text-transform: uppercase;
+    user-select: none;
+    vertical-align: middle;
+    white-space: nowrap;
+
+    @include media-breakpoint-up(md) {
+      font-size: $btn-font-size-md;
+    }
+
+    // Share hover and focus styles
+    @include hover-focus {
+      text-decoration: none;
+    }
+
+    &:hover,
+    &.active,
+    &:active,
+    &.focus,
+    &:focus {
+      border-color: transparent;
+    }
+
+    &:focus,
+    &.focus {
+      box-shadow: $btn-focus-box-shadow;
+      color: initial;
+      outline: 0;
+    }
+
+    // Disabled comes first so active can properly restyle
+    &.disabled,
+    &:disabled {
+      @include box-shadow(none);
+      cursor: $cursor-disabled;
+      opacity: .65;
+    }
+
+    &:active,
+    &.active {
+      @include box-shadow($btn-focus-box-shadow, $btn-active-box-shadow);
+      background-image: none;
+    }
+  }
+
+  .btn-primary {
+    @include button-variant($btn-primary-color, $btn-primary-bg, $btn-primary-border);
+  }
+
+}
+
+// Horizontal spotlight layout
+.spotlight-horizontal {
```

Comment:
Horizontal spotlights don't have background-size, position, or repeat rules assigned to them